ta/arch/arm/user_ta_header.c

```diff
@@ -57,7 +57,14 @@ void __utee_entry(unsigned long func, unsigned long session_id,
 const struct ta_head ta_head __section(".ta_head") = {
 	/* UUID, unique to each TA */
 	.uuid = TA_UUID,
-	.stack_size = TA_STACK_SIZE,
+	/*
+	 * According to GP Internal API, TA_STACK_SIZE corresponds to the stack
+	 * size used by the TA code itself and does not include stack space
+	 * possibly used by the Trusted Core Framework.
+	 * Hence, stack_size which is the size of the stack to use,
+	 * must be enlarged
+	 */
+	.stack_size = TA_STACK_SIZE + 1152,
```

Comment:
Agree, but I'd also like to know what these `1152` bytes are reserved for. I know we use some space for the `struct tee_ta_param` which is passed to the entry function. We could have a bug lurking.